web/annotation_layer_builder.css

```diff
@@ -52,6 +53,19 @@
   width: 100%;
 }
 
+.annotationLayer .textWidgetAnnotation input[disabled],
+.annotationLayer .textWidgetAnnotation textarea[disabled] {
+  background: none;
+  border: 1px solid transparent;
+  cursor: not-allowed;
+}
+
+.annotationLayer .textWidgetAnnotation textarea {
```

Comment:
Shouldn't a `textarea` have the same `hover`/`focus` rules as a normal `input`!?
Because currently the background colour isn't removed when editing its contents, and there's no outline on either.